src/controllers/artist.js

```diff
@@ -0,0 +1,125 @@
+const db = require('../db/index');
+
+const createArtist = async (req, res) => {
+  const { name, genre } = req.body;
+
+  try {
+    const {
+      rows: [artist],
+    } = await db.query(
+      'INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *',
+      [name, genre]
+    );
```

Comment:
Minor point but you could access the name and genre values directly in the function without destructuring them first, as you are only using them once:
```
  try {
    const {
      rows: [artist],
    } = await db.query(
      'INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *',
      [req.body.name, req.body.genre]
    );
```